docs/wiki/What-is-Enterprise-Scale.md

```diff
@@ -33,7 +33,11 @@ The following table outlines key customer requirements in terms of landing zones
 
 ## Pricing
 
-There’s no cost associated with Enterprise-Scale, and you only pay for the Azure services that are being enabled, and the services your organization will deploy into the landing zones. For example, you don’t pay for the management groups or the policies that are being assigned, but policy to enable Azure Security Center on the landing zone subscriptions will generate cost on those subscriptions.
+There’s no cost associated with Enterprise-Scale itself, as it is just an architecture that is constructed using existing Azure products and services. Therefore you only pay for the Azure products and services that you choose to enable, and also the products and services your organization will deploy into the various application/service landing zones for your workloads.
+
+For example, you don’t pay for the Management Groups or the Azure Policies that are being assigned, but assigning a policy to enable Azure Defender (previously known as Azure Security Center Standard) on all landing zone subscriptions will generate cost on those subscriptions for the Azure Defender service as detailed [here](https://azure.microsoft.com/pricing/details/azure-defender/).
```

Comment:
Maybe update to explain that the cost is only incurred for resources for which Azure Defender is enabled in those Subscriptions, as the process of linking Azure Defender to a Subscription alone doesn't incur cost?

I still think we should also highlight some of the products which incur a "more significant" consumption cost, such as DDOS Protection Standard, Azure Firewall, VPN Gateway, etc.

Reply:
Thanks Kevin. 

Let's see what others have to say also before making a change. I feel with links to the pricing calculator and calling out that you determine the resources as part of the design areas this is covered, without providing and maintaining a full list of resources that could have a cost (of any size).

@victorar, @krnese, @mbilalamjad - thoughts?

Reply:
checkout the latest commit @krowlandson and let me know